lineman/app/components/edit_group_form/edit_group_form.coffee

```diff
@@ -6,9 +6,14 @@ angular.module('loomioApp').factory 'EditGroupForm', ->
     $scope.submit = ->
       $scope.group.save().then ->
         $scope.$close()
-        FlashService.success 'group_form.edit_group_success'
-      , ->
-        $rootScope.$broadcast 'pageError', 'cantEditGroup', $scope.group
+        FlashService.success 'edit_group_form.messages.success'
```

Comment:
Time to start thinking about how we're going to generalize this pattern, as I have a feeling we're about to implement 10x over the next week or two.

I was thinking
```
(response) ->
  ErrorService.applyError response, $scope.group
```
```
ErrorService ($rootScope) ->
  applyError: (response, model) ->
    case response.status
      when 422 then $scope.group.setErrors response.data.errors
      when 400 then @broadcastError 'badRequest', response
      when 401, 403 then @broadcastError 'requestForbidden', response
      when 404 then @broadcastError 'connectivityLost', response
      when 500 then @broadcastError 'serverError', response
    
  broadcastError: (errorType, response) ->
    $rootScope.$broadcast errorType, response
```